### ISSUE TITLE
Shortcuts in menu bar prevent searching for components

### ISSUE THREAD

**Issue by PPillau** [gluonhq/scenebuilder#250]
The single-key-shortcuts for the items in the menu bar (F -> File, E -> Edit, V -> View...) prevent me from actually typing something in the library search bar. I am always tabbing into the respective menu when trying to search some component. 
There should be a way to change keyboard shortcuts or turn these of completely or just not make them respond when typing in the search bar for example because right now search doesn't seem possible...

**Comment by DezzardHD** [comment]
I've encountered the same problem. Did you find a solution for this issue?

**Comment by safirex** [comment]
this isn't single-key-shortcut actually, it's the accelerators you activate whenever you press alt (when alt+tab for exemple) and stays activated unless you re-press tab or echap  but true it's pretty annoying when it acts up
One way to deal with it would be to need alt to stay pushed for the accelarator to be enabled (alt+F, alt+E instead of alt,F,E)

**Comment by tharindusathis** [comment]
I face the same issue. When I type something, scene builder thinks I have pressed 'alt' key at random times.

**Comment by DezzardHD** [comment]
> I face the same issue. When I type something, scene builder thinks I have pressed 'alt' key at random times.

I found myself switching between windows using ALT+Tab. This was my problem that I've solved by just typing ALT when I tried to write something, but instead a menu from the menubar opened.
I don't know, why this could be a "random" behavior.

**Comment by tharindusathis** [comment]
Yes, I also frequently use ALT+Tab. That may be a trigger for the above problem.

**Comment by FeixuRuins** [comment]
Any update?

**Comment by Dalikey** [comment]
> Any update?

Just press alt once and fixed.

**Comment by geekarist** [comment]
It seems related to the following JDK bugs:
- [[JDK-8090647] Mnemonics : on windows we should cancel the underscore latch when an app loses focus. - Java Bug System](https://bugs.openjdk.java.net/browse/JDK-8090647)
- [[JDK-8119000] Menu mnemonics 'stick' on Windows 7 - Java Bug System](https://bugs.openjdk.java.net/browse/JDK-8119000)

**Comment by iKatsuo** [comment]
Any updates about it?
I'm using the latest version of SceneBuilder 18.0.0 and I'm still having the same issue...

**Comment by Snowfox07** [comment]
pressing ALT key will temporarily fix it

**Comment by Oliver-Loeffler** [comment]
Well, Scene Builder here exactly behaves like other Windows applications (like Pint, Notepad, Word etc.).
1x pressing ALT makes the menubar keyboard navigatable. Of course, this collides with search for components. So this can be clumsy.

Following behavior could help.
1. focus at search text field - `ALT` is ignored
2. when pressing `ESC` or `TAB` focus is moved away from text field and ALT is continued to be accepted

What do you think?

**Comment by MMDPROJECT** [comment]
> this isn't single-key-shortcut actually, it's the accelerators you activate whenever you press alt (when alt+tab for exemple) and stays activated unless you re-press alt or echap but true it's pretty annoying when it acts up One way to deal with it would be to need alt to stay pushed for the accelarator to be enabled (alt+F, alt+E instead of alt,F,E)

@safirex God bless you my brother (خداوند تورا بلیسد برادر من) Problem got solved

**Comment by Walid-X** [comment]
when i want to search for something i just keep holding alt + what i want to type , it might be annoying but with time u might doing it unconsciously.

**Comment by Sash-J** [comment]
Just Press the ALT key. It will activate or disable the hotkeys for the menu stripe 👍

**Comment by seujorgenochurras** [comment]
Dude why is that a feature?
And even worse why I can't disable it???
Is it that hard to implement
